ComponentKit/Core/Scope/CKComponentScopeFrame.h

```diff
@@ -34,4 +34,16 @@ struct CKComponentScopeFramePair {
 
 @property (nonatomic, strong, readonly) CKComponentScopeHandle *handle;
 
+struct _CKChildScopeFrameRet {
+  Class cls;
+  id identifier;
+  CKComponentScopeFrame *frame;
+};
+
+- (_CKChildScopeFrameRet)childScopeFrameForOldIdentifier:(CKComponentScopeHandleIdentifier)globalIdentifier;
```

Comment:
`childScopeHierarchyForScopeFrameToken:(id)scopeFrameToken`

Inside the implementation just check if the scope frame token is an NSNumber and unwrap it if so. Kinda sucks as it's a hidden coupling with CKComponent's `scopeFrameToken`, but it is better than exposing another method on CKComponent (even if it's internal-only)